lib/widgets/card.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:flutter/material.dart';
+
+class CustomCard extends StatelessWidget {
+  final String text;
+  final String image;
+  final VoidCallback onClick;
+  const CustomCard(
+      {Key? key,
+      required this.text,
+      required this.image,
+      required this.onClick})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
```

Comment:
I'm not sure how it will pan out, but I would love to have a background color, possibly gray, giving the card an outline. The user may not understand if they have to click the icons in the current setup. You'll share the screenshots and so I can make a judgement on whether to proceed with the background color.,

Reply:
check whatsapp for screenshots and compare

Reply:
I have shared with you a folder with screens for this particular section, we'll go with the suggested landing page designs therein.

Reply:
Here is [the link](https://drive.google.com/file/d/1t7w3J6bvsOtMWhH6O_UkmQg2g5ze_tsr/view?usp=sharing) to the landing screen in the Google Drive folder.